predict.py

```diff
@@ -150,6 +150,15 @@ def predict_one_job(
         " REVIEW, PREDICTION), defaults to 'DEFAULT,REVIEW'"
     ),
 )
+@click.option(
+    "--target-jobs",
+    default=None,
+    multiple=True,
+    help=(
+        "Comma separated list of target jobs to train on. "
```

Comment:
Add a specific target job for which to output the predictions (multiple can be passed if --target-job <job_name> is repeated)